src/examples/confgure_npm_for_heroku.yml

```
@@ -0,0 +1,25 @@
+description: >
+  Configure npm for a Heroku nodejs applicaton and install deploy to Heroku.
+usage:
+  version: 2.1
+  orbs:
+    cloudsmith-npm: ft-circleci-orbs/cloudsmith-npm@1.1
+  jobs:
+    build:
+      docker:
+        - image: cimg/node:20.12
+      steps:
+        - checkout
+        - cloudsmith-npm/configure_npm_for_heroku:
+            repository: "your-repository-id"
+            app_name: "your-heroku-app-name"
+        - run:
+            name: Deploy to Heroku
+            command: |
+              REMOTE=git.heroku.com/heroku-cloudsmith.git
```

Is heroku-cloudsmith what users would use or do we need to make this obvsiouly a token that needs replacing?

---

Good question, this is only the deploy method that we're using in the example, but if others have it set up a different way this part of the example is interchangeable. I can make this obvious but every team who's currently deploying to heroku will see that this is the deploy step

---

Good spot. We should change this to be a generic heroku app name. Most people have this in their config:
```suggestion
              REMOTE=git.heroku.com/<<parameters.app-name>>.git
```